packages/account/src/utils/Constants.ts

```diff
@@ -31,19 +31,20 @@ export const BICONOMY_IMPLEMENTATION_ADDRESSES: BiconomyImplementations = {
   '0xf1080f5f874ea8170e423738791e0e9a8aad87fd': 'V2_0_0'
 }
 
-export const ENTRYPOINT_ADDRESSES_BY_VERSION: EntrypointAddressesByVersion = {
-  'V0_0_5': '0x27a4db290b89ae3373ce4313cbeae72112ae7da9',
-  'V0_0_6': '0x5ff137d4b0fdcd49dca30c7cf57e578a026d2789'
+export const ENTRYPOINT_ADDRESSES_VERSIONED: ENTRY_POINT_ADDRESSES_BY_VERSION = {
```

Comment:
I think you misunderstood my point.

i was talking about only variable and constants name, not the type names. Type name are fine but for example instead of `EntrypointAddressesByVersion` we should use `EntryPointAddressesByVersion` with capital P.